tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import os
+import numpy as np
+import boto3
+import pytest
+from moto import mock_aws
+
+
+@pytest.fixture(scope="module")
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
```

Comment:
Prefer to use `monkeypatch.setenv` so that the env is cleaned up after test run. Also might not need this at all if we're using moto? Does it error without?